app/wasm_filter_test.go

```diff
@@ -0,0 +1,604 @@
+package app
+
+import (
+	"testing"
+
+	ibctransfertypes "github.com/cosmos/ibc-go/v4/modules/apps/transfer/types"
+	clienttypes "github.com/cosmos/ibc-go/v4/modules/core/02-client/types"
+	channeltypes "github.com/cosmos/ibc-go/v4/modules/core/04-channel/types"
+	"github.com/golang/protobuf/proto" // nolint: staticcheck
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	codectypes "github.com/Finschia/finschia-sdk/codec/types"
+	sdk "github.com/Finschia/finschia-sdk/types"
+	banktypes "github.com/Finschia/finschia-sdk/x/bank/types"
+	distributiontypes "github.com/Finschia/finschia-sdk/x/distribution/types"
+	govtypes "github.com/Finschia/finschia-sdk/x/gov/types"
+	stakingtypes "github.com/Finschia/finschia-sdk/x/staking/types"
+	wasmkeeper "github.com/Finschia/wasmd/x/wasm/keeper"
+	"github.com/Finschia/wasmd/x/wasm/keeper/wasmtesting"
+	wasmtypes "github.com/Finschia/wasmd/x/wasm/types"
+	wasmvmtypes "github.com/Finschia/wasmvm/types"
+)
+
+func TestFilteredStargateMsgEncoders(t *testing.T) {
```

Comment:
`wasmFilteredEncodeStargateMsg` just wrap `StargateEncoder`. So I think we just need to verify if this change can block `"/lbm.fswap.v1", "/lbm.fbridge.v1"` instead of writing duplicate test cases

Reply:
But I want to check if other tasks also work properly with this encoder change.

Reply:
Then, I think you can use `Stargate` part instead of writing whole test cases.

Reply:
OK, good.